README.md

```diff
@@ -0,0 +1,29 @@
+# SMT String Bench Results
+
+## Running the experimental evaluation
+
+1. Set up the Python virtual environment:
+```sh
+python -m venv .venv
+pip install -r requirements.txt
+source .venv/bin/activate
+```
+
+2. Get new tasks from the server with experimental results:
+```sh
+./get_tasks_and_generate_csv.sh
+```
+
+3. Process the results:
+```sh
+./pyco_proc.py [options] <requested_tasks_file_with_results.tasks>
+```
```

Comment:
Why is this step here? This should be done in `get_tasks_and_generate_csv.sh`

Reply:
It should be done in `eval.py`, not in `get_tasks_and_generate_csv.sh`.

Reply:
True. I removed this step.